src/ResourceManagement/CosmosDB/IndexingPolicyImpl.cs

```diff
@@ -0,0 +1,278 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.CosmosDB.Fluent.Models;
+using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Management.CosmosDB.Fluent
+{
+
+    internal partial class IndexingPolicyImpl<ParentImplT, IParentT, DefinitionParentT, UpdateParentT> :
+        ChildResource<Models.IndexingPolicy, ParentImplT, IParentT>,
```

Comment:
Make sure this is not leaked to user. This one appears not actually a Resource, it does not have id or name.

Reply:
done, use wrapper instead. And actually it has no interface, thus it would not be leaked to user.